### PR TITLE
feat(led-545): HPOS compatibility

### PR DESCRIPTION
**Pull request by timjackleus**

### CODE REVIEW

File: classes/class-ledyer-main.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Ledyer_Checkout_For_WooCommerce {
 	 */
 	public $checkout;
 
-	const VERSION = '1.7.8';
+	const VERSION = '1.8.0';
```
Comment (perjo927) [inline classes/class-ledyer-main.php:54]: not sure if versioning is handled by CI

File: classes/class-ledyer-singleton.php [hunk 2]
```diff
@@ -21,12 +21,12 @@ trait Singleton {
...
 	private static $settings = array();
 
-	/**
-	 * Instanciate the class
-	 *
-	 * @return self
-	 */
-	public static function instance() {
+  /**
+   * Instantiate the class.
```
Comment (perjo927) [inline classes/class-ledyer-singleton.php:25]: nice

**Review by perjo927 — APPROVED** [review]
LGTM!

File: classes/class-ledyer-main.php (hunk 1 above)
Comment (timjackleus) [reply classes/class-ledyer-main.php:54]: Good point I've removed the CI for now as it's not helping

**Pull request merged**